Sources/GravatarUI/SwiftUI/AvatarPicker/AvatarPickerViewModel.swift

```diff
@@ -26,6 +26,7 @@ class AvatarPickerViewModel: ObservableObject {
     }
 
     @Published var selectedAvatarURL: URL?
+    @Published var backendSelectedAvatarURL: URL?
```

Comment:
Can we make this read only outside the class?  It compiles.  And I think this is the only class that would set it.

```suggestion
    @Published private(set) var backendSelectedAvatarURL: URL?
```